static/js/scanners/scannerresult_actions.js

```diff
@@ -1,7 +0,0 @@
-document.querySelectorAll('.report-false-positive-button').forEach((button) => {
-  button.addEventListener('click', () => {
-    setTimeout(() => {
-      window.location.reload();
```

Comment:
I've not reviewed this code before, but isn't this js to allow the page to reflect the updated result?  Is that not needed now too?  (the issue didn't mention about page reloads)

Reply:
The patch is changing the form target from "_blank" (opening a new tab) "_self"  (default, unspecified), which will cause the original tab to be reloaded.